<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grafika</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFunctions">
    <property name="title">
     <string>&amp;Functions</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionFuntionSyntax"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionRussian"/>
    </widget>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="menuThemes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFunctions"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionFuntionSyntax">
   <property name="text">
    <string>Funtion's &amp;syntax</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRussian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Russian</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
